OpenEdXMobile/src/main/java/org/edx/mobile/module/prefs/LoginPrefs.java

```diff
@@ -165,14 +165,14 @@ public ProfileImage getProfileImage() {
     }
 
     @NonNull
-    private static String segmentKeyFromAuthBackend(@NonNull AuthBackend backend) {
+    private static String eventKeyFromAuthBackend(@NonNull AuthBackend backend) {
```

Comment:
Maybe change 'eventKey' to analyticsKey'? Or maybe "analyticsToken', since it technically is a value, not key.